help/templates/template.cshtml

```diff
@@ -200,6 +200,7 @@
 																<li><a href="/apidocs/v5/index.html#Fake.Tools.Git">Git</a></li>
 																<li><a href="/fake-tools-pickles.html">Pickles</a></li>
 																<li><a href="/fake-tools-octo.html">Octo</a></li>
+                                                                <li><a href="/fake-tools-squirrel.html">Octo</a></li>
```

Comment:
I guess you mean `Squirrel` here not `Octo` :)

Reply:
XD